data/data.py

```diff
@@ -0,0 +1,154 @@
+from datetime import datetime, timedelta
+from final import Final
+from pregame import Pregame
+from scoreboard import Scoreboard
+import mlbgame
+import debug
+import time
+
+try:
+    from urllib.error import URLError
+except ImportError:
+    from urllib2 import URLError
+
+NETWORK_RETRY_SLEEP_TIME = 5.0
+
+class Data:
+  def __init__(self, config):
+    # Save the parsed config
+    self.config = config
+
+    # Parse today's date and see if we should use today or yesterday
+    self.year, self.month, self.day = self.__parse_today()
+    # self.year, self.month, self.day = (2018, 4, 22)
+
+    # Flag to determine when to refresh data
+    self.needs_refresh = True
+    
+    # Fetch the games for today
+    self.refresh_games()
+
+    # Fetch all standings data for today
+    # (Good to have in case we add a standings screen while rotating scores)
+    self.refresh_standings()
+
+    # What game do we want to start on?
+    self.current_game_index = self.game_index_for_preferred_team()
+
+
+  #
```

Comment:
Could you clean up this area there's some extra whitespace and some unused comments